pkg/cli/session/session.go

```diff
@@ -53,10 +52,6 @@ func (e *BackplaneSession) RunCommand(cmd *cobra.Command, args []string) error {
 		e.Options.Alias = args[0]
 	}
 	if e.Options.ClusterId == "" && e.Options.Alias == "" {
```

Comment:
As we discussed, it should be better if we can validate the cluster id when setup the session.
If it is not going to be addressed in this PR, could you help create a card to track?

Reply:
Check for valid cluster keys